.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+**Organization**:
```

Comment:
[GitHub documentation](https://help.github.com/articles/creating-an-issue-template-for-your-repository/) says this needs to be in the default branch, so this is in preparation for changing the default to nist-pages.